src/github.com/getlantern/lantern/feed.go

```diff
@@ -0,0 +1,170 @@
+package lantern
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"math"
+	"net/http"
+	"strings"
+
+	"github.com/getlantern/eventual"
+	"github.com/getlantern/flashlight/util"
```

Comment:
That said, I guess that makes packaging for Android more complicated, so never mind.